perllib/FixMyStreet/App/Controller/Admin/Triage.pm

```diff
@@ -111,6 +111,14 @@ sub update : Private {
     my $current_category = $problem->category;
     my $new_category = $c->get_param('category');
 
+    if (!$new_category) {
+        my $errors = $c->stash->{errors} || [];
+        push @$errors, _"Please choose a category";
+        $c->stash->{errors} = $errors;
+
+        $c->detach('/report/id', [ $problem->id ] );
```

Comment:
I see this appears to work, but I think it's a bit confusing to go back to this (and it only not infinite-loop because it's the midpoint of a chain); would a $c->detach just to stop processing here not do the same job? (Note, I haven't actually checked this, sorry)